api/files/serializers.py

```diff
@@ -303,10 +303,14 @@ def get_absolute_url(self, obj):
 
 
 class FileSerializer(BaseFileSerializer):
-    node = RelationshipField(related_view='nodes:node-detail',
-                             related_view_kwargs={'node_id': '<node._id>'},
-                             help_text='The project that this file belongs to'
-                             )
+
+    target = TargetField(link_type='related', meta={'type': 'get_target_type'})
```

Comment:
Looks like.

Reply:
I think adding in the target and not removing the node is the safest way, and we can consider removing the node attribute in a future version.

Reply:
But since the relationship no longer exists on the model:

```
node = ShowIfVersion(
    ser.SerializerMethodField(),  # returns .target iff it's a node-type
    min_version='2.0', max_version=<current version>
)
```